src/interface/blas3/backend/amd_gpu.hpp

```diff
@@ -123,18 +131,79 @@ _gemm(sb_handle_t& sb_handle, index_t _M, index_t _N, index_t _K,
       }
     } else
 #endif  // GEMM_TALL_SKINNY_SUPPORT
-      if (_M * _N <= 65536) {
+      // Folling configuration are taken using the auto tuner on amd-mi210
+      // and divided following their arith_intensity or another ratio between _N
+      // and _K input size
+      if (arith_intensity >= 360 || (_N >> 4) > _K) {
```

Comment:
The `arithm_intensity` based if/else statements seem coherent, but combining them with the `(_N >> 4) > _K)` confuses me a little bit. If it has a direct impact _(independently or depending on the combination's `arithm_intensity`)_, isn't it better to put is a stand-alone if/else statement ? 
Something like : 
```
if( (_N >> 4) > _K))
// Config or Configs _(depending on arith_intensity - 2 in total if we follow the setup here : arith_intensity > or <= 100)_
else {
// Rest of `arith_intensity` conditional statements.
}
```

Reply:
Thank you for your review @OuadiElfarouki 
I addressed it in commit 94d5704. Please let me know if it seems more readable to you now, otherwise I can revert it back to what it was.
The performance are not affected by this change.

Reply:
Yes looks more readable now thanks a lot!